examples/cucumber/.sauceignore

```diff
@@ -0,0 +1,12 @@
+# This file instructs saucectl to not package any files mentioned here.
+.git/
+.github/
+.DS_Store
+.hg/
+.vscode/
+.idea/
+.gitignore
+.hgignore
+.gitlab-ci.yml
+.npmrc
+*.gif
```

Comment:
@valentyn88 having .saucecignore doesnt affect at all for espresso. I'd remove it to avoid confusion.

Reply:
I remove this file